covid19_sfbayarea/data/san_francisco.py

```diff
@@ -168,67 +173,65 @@ def get_age_table(session : SocrataApi, resource_ids: Dict[str, str]) -> List[Di
 def get_gender_table(session : SocrataApi, resource_ids: Dict[str, str]) -> Dict:
     """Get cases by gender"""
 
-   # Dict of source_label:target_label for re-keying.
+    # Dict of source_label:target_label for re-keying.
+    # Note: non cis genders not currently reported
     resource_id = resource_ids['gender']
-    GENDER_KEYS = {"Female": "female", "Male": "male", "Unknown": "unknown"}
-    params = {'$select': 'gender, MAX(cumulative_confirmed_cases) as cumul_cases',
-            '$group': 'gender' }
-    data = session.resource(resource_id, params=params)
-
-    # check that the set of genders present in data is equal to the genders we expect
-    data_genders  = set( [entry['gender'] for entry in data])
-    if GENDER_KEYS.keys() != data_genders:
-        raise FormatError(
-            f"Did not get expected Gender categories. Data received: {data}")
-
-    # re-key
-    gender_table = {GENDER_KEYS[entry["gender"]]: int(
-        entry["cumul_cases"]) for entry in data}
-
-    return gender_table
+    GENDER_KEYS = {"Female": "female", "Male": "male", "Unknown": "unknown", "Trans Female": "female", "Trans Male": "male"}
+    # find the latest date of data collection
+    params = {'$select': 'max(specimen_collection_date) as date'}
+    latest_date = session.resource(resource_id, params=params)[0]
 
-def get_transmission_table(session : SocrataApi, resource_ids: Dict[str, str]) -> Dict:
-    """Get cases by transmission category"""
-    resource_id = resource_ids['cases_deaths_transmission']
-    # Dict of source_label:target_label for re-keying
-    TRANSMISSION_KEYS = {"Community": "community",
-                        "From Contact": "from_contact", "Unknown": "unknown"}
-    params = { '$select': 'transmission_category, sum(case_count)', '$group': 'transmission_category'}
+    # get cumulative confirmed cases on latest date
+    params = {'$select': 'gender, cumulative_confirmed_cases as cases', '$where':f'specimen_collection_date="{latest_date["date"]}"'}
     data = session.resource(resource_id, params=params)
+
     # re-key
-    transmission_data = { TRANSMISSION_KEYS[ entry["transmission_category"] ]: int(entry["sum_case_count"]) for entry in data}
-    return transmission_data
+    table : Dict[str, int]= dict()
+    for entry in data:
+        rekey = GENDER_KEYS[entry['gender']]
+        table[rekey] = table.get(rekey,0) + int(entry["cases"])
```

Comment:
This threw me for a minute, making me think you were working around some case where the source data had repeated genders, but then I realized it’s because we are combining some of the genders from the source data. It would probably be useful to have a comment here explaining that’s why you’re doing this otherwise-slightly-obtuse-looking operation.